lib/init/grass7.html

```diff
@@ -461,11 +478,10 @@ <h2>SEE ALSO</h2>
 List of <a href="variables.html">GRASS environment variables</a>
 
 <p>
-<a href="http://grass.osgeo.org">GRASS GIS Web site</a><br>
-<a href="http://grass.osgeo.org/wiki/">GRASS GIS User Wiki</a><br>
-<a href="http://trac.osgeo.org/grass/">GRASS GIS Bug Tracker</a><br>
```

Comment:
Are these the only places with old URLs?

Reply:
The only ones (except for one in r.in.mat.html)

Reply:
r.in.mat.html fixed in 53cd8eb